.github/workflows/avm.res.container-service.managed-cluster.yml

```diff
@@ -0,0 +1,81 @@
+name: "avm.res.container-service.managed-cluster"
```

Comment:
Don't forget to uncomment the line in the codeowners file specific to your module path